qemu/tests/cfg/commit_with_bitmaps_nospace.cfg

```diff
@@ -0,0 +1,28 @@
+- commit_with_bitmaps_nospace:
```

Comment:
Do we want to run this test with network storage backend?

Reply:
@zhencliu , test it just for local backend.